### PR TITLE
fix: build main branch

### PR DESCRIPTION
**Pull request by lauthieb**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Make sure you are making a pull request against the **main branch** (left side).
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Description

Hotfix this: https://github.com/Decathlon/vitamin-web/actions/runs/477293693/workflow

## Does this introduce a breaking change?

- No

### CODE REVIEW

**Review by jleveugle — APPROVED** [review]

**Review by gabbloquet — APPROVED** [review]
Lgtm :)

File: packages/sources/css/postcss.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const postCssImport = require('postcss-import');
+const postcssColorMod = require('postcss-color-mod-function');
```
Comment (NicolasBreunin) [inline packages/sources/css/postcss.config.js:2]: postCssImport & postCssColorMod  or  postcssImport & postcssColorMod ?

Comment (lauthieb) [reply packages/sources/css/postcss.config.js:2]: I don't understand your question Nicolas?

Comment (gabbloquet) [reply packages/sources/css/postcss.config.js:2]: Uppercase on C => postCssColorMod
```suggestion
const postCssColorMod = require('postcss-color-mod-function');
```
Maybe @lauthieb

Comment (lauthieb) [reply packages/sources/css/postcss.config.js:2]: The convention seen in several documentation is `postcss` without uppercase.

Comment (NicolasBreunin) [reply packages/sources/css/postcss.config.js:2]: yes sorry if my comment was not clear

Comment (NicolasBreunin) [reply packages/sources/css/postcss.config.js:2]: in this case you can correct by `const postcssImport = require('postcss-import');`

Comment (lauthieb) [reply packages/sources/css/postcss.config.js:2]: Yes, thanks!

Comment (lauthieb) [reply packages/sources/css/postcss.config.js:2]: Resolved.

**Pull request merged**